frontend/src/modules/layout/components/menu/menu-header.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <div
+    class="h-14 flex items-center transition group pl-5 pr-3"
+  >
+    <router-link to="/">
+      <img
+        key="logo"
+        class="h-6 !opacity-100"
+        src="/images/logo/crowd-mini.svg"
+        alt="crowd.dev logo"
+      />
+    </router-link>
+    <el-button
+      class="absolute btn btn--icon--sm btn--transparent !h-8 w-8 flex items-center transition-none"
+      :class="props.collapsed ? 'right-4 opacity-0 hover:opacity-100' : 'right-3'"
+      @click="emit('toggle-menu')"
+    >
```

Comment:
When closing the menu there's a slight delay for the button to disappear. Could you improve that so that it's a bit smoother?

Reply:
Fixed